notebooks/en/index.md

```diff
@@ -27,6 +27,7 @@ Check out the recently added notebooks:
 - [Multimodal Retrieval-Augmented Generation (RAG) with Document Retrieval (ColPali) and Vision Language Models (VLMs)](multimodal_rag_using_document_retrieval_and_vlms)
 - [Fine-Tuning a Vision Language Model (Qwen2-VL-7B) with the Hugging Face Ecosystem (TRL)](fine_tuning_vlm_trl)
 - [Multi-agent RAG System 🤖🤝🤖](multiagent_rag_system)
+- [Multimodal RAG with ColQwen2, Reranker, and Quantized VLMs on Consumer GPUs](multimodal_rag_using_document_retrieval_and_reranker_and_vlms)
```

Comment:
If you don't mind, we could also edit `index.md` to display only the last 5 recipes to avoid cluttering the main page.